python/python/glide/async_commands/core.py

```diff
@@ -2930,6 +2930,43 @@ async def zremrangebylex(
             ),
         )
 
+    async def zremrangebyrank(
+        self,
+        key: str,
+        start: int,
+        end: int,
+    ) -> int:
+        """
+        Removes all elements in the sorted set stored at `key` with rank between `start` and `end`.
+        Both `start` and `end` are zero-based indexes with `0` being the element with the lowest score.
```

Comment:
```suggestion
        Both `start` and `end` are zero-based indexes with 0 being the element with the lowest score.
```
in python we dont do like this, see lrange